apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.ts

```diff
@@ -0,0 +1,9 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-card-list-example',
+    templateUrl: 'card-list-example.component.html',
+    styleUrls: ['./card-list-example.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
I don't think you need OnPush in this case. You have no component logic here.

Reply:
removed OnPush ChangeDetectionStrategy